### ISSUE TITLE
Extension does not exist on the Chrome store (404)

### ISSUE THREAD

**Issue by andreoliwa** [chitsaou/copy-as-markdown#84]
## Summary

First of all, thank you for this extension, it is awesome, the best one in the category.

The Chrome extension seems to be gone from the Chrome store.
The download link in the description is not working.

## Reproduction Steps

1. Click on the [Chrome Extension URL](https://chrome.google.com/webstore/detail/copy-as-markdown/fkeaekngjflipcockcnpobkpbbfbhmdn?hl=en) from the [Download](https://github.com/chitsaou/copy-as-markdown#download) section of the README.
2. "Error 404 (not found)".

## Expected Behavior

Open the extension page on the Chrome Web Store.

## Actual Behavior

"Error 404 (not found)".

## Reproducible Environment

- Browser (Version): Brave Dev Version 1.10.84 Chromium: 81.0.4044.138 (Official Build) dev (64-bit)
- Operating System (Version): macOS Catalina 10.15.5
- System Language: English, en_US

**Comment by zbjdonald** [comment]
Same Issue

**Comment by andreoliwa** [comment]
A temporary solution I found:

1. Open [Download Copy as Markdown 2.4.3 CRX File for Chrome - Crx4Chrome](https://www.crx4chrome.com/crx/3417/).
2. Click on the ["Download crx file from Crx4Chrome >" URL](https://www.crx4chrome.com/go.php?p=3417&s=1&l=https%3A%2F%2Ff2.crx4chrome.com%2Fcrx.php%3Fi%3Dfkeaekngjflipcockcnpobkpbbfbhmdn%26v%3D2.4.3).
3. Open Chrome or [Brave](brave://extensions/) extensions page.
4. Select "Developer mode".
5. Refresh the extensions page (I have to do that on Brave, it doesn't work without a refresh 🤷‍♂️ ).
6. Drag the downloaded `.crx` file onto the extensions page and click "Install".

**Comment by chitsaou** [comment]
Hi everyone,

Sorry for late response.

I am aware that this extension has been pulled down from Chrome Web Store. It is because this extension is violation the new policy:

### User Data Privacy

Your product violates the "Use of Permissions" section of the policy, which requires that you:

- Request access to the narrowest permissions necessary to implement your product’s features or services.
- If more than one permission could be used to implement a feature, you must request those with the least access to data or functionality.
- Don't attempt to "future proof" your product by requesting a permission that might benefit services or features that have not yet been implemented.

---

I've tried to fix that by removing unnecessary permissions and even some features. Unfortunately it still didn't pass the review.

I tried to appeal but the support team refused to give me more technical details.

I don't have any idea to proceed anymore. I think this extension will not be available on Chrome Web Store unless the policy is changed.

Meanwhile I am considering an alternative way to distribute the extension, but that requires building an auto update system. I may not be able to make it happen in a reasonable time.

Sorry about that.

**Comment by chitsaou** [comment]
@andreoliwa I appreciate your help, but please be aware that any extension downloaded from third-party website may contain malicious code. 

You can get an authentic version by downloading the zip from repository page directly, and follow README.md for developer mode.

**Comment by andreoliwa** [comment]
@chitsaou Whoops, I didn't know that website could be malicious, thank you.
Last time I gave up because I didn't want to `npm install` the project, so I searched the web for a `.crx` file, and I didn't know how to use the `.zip`.

Now I downloaded the zip file as you said, unpacked it, and then added it to the browser.
It's working fine. 😉

I started from the instructions in the [Debugging](https://github.com/chitsaou/copy-as-markdown#debugging) section of the README.
I wrote more detailed steps in the README: https://github.com/chitsaou/copy-as-markdown/pull/85
It was not 100% clear for me how to install it, maybe other people will have the same doubts.

Thanks again!
